tasks/collect-data/collect-data.yaml

```diff
@@ -69,6 +69,9 @@ spec:
     - name: snapshotNamespace
       type: string
       description: namespace where Snapshot is located
+    - name: releasePipelineMetadata
```

Comment:
Is this going to end up as a pipeline result? I am not sure anyone looks as task results

Reply:
This helps with unit tests. In addition, if ever we loose the log messages (tekton results issue), we will have this result stored as part of the TaskRun.